medallion/backends/base.py

```diff
@@ -26,6 +45,48 @@ def iter_(mcls):
 
 class Backend(object, metaclass=BackendRegistry):
 
+    def __init__(self, **kwargs):
+        self.next = {}
+
+        if kwargs.get("run_cleanup_threads", True):
+            self.timeout = kwargs.get("session_timeout", 30)
+            checker = TaskChecker(kwargs.get("check_interval", 10), self._pop_expired_sessions)
+            checker.start()
+
+            self.status_retention = kwargs.get("status_retention", SECONDS_IN_24_HOURS)
+            if self.status_retention != -1:
+                if self.status_retention < SECONDS_IN_24_HOURS and get_application_instance_config_values(APPLICATION_INSTANCE,
+                                                                                                          "backend",
+                                                                                                          "interop_requirements"):
```

Comment:
Could pull the `get_application_instance_config_values` result into a variable and reuse it down in the if statement on line 66, for readability.

Reply:
done